Riot/Modules/Room/Views/BubbleCells/Styles/Bubble/Cells/FileWithoutThumbnail/Incoming/FileWithoutThumbnailIncomingBubbleCell.swift

```diff
@@ -0,0 +1,39 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+
+class FileWithoutThumbnailIncomingBubbleCell: FileWithoutThumbnailBaseBubbleCell {
+    
+    override func setupViews() {
+        super.setupViews()
+        
+        bubbleCellContentView?.showSenderInfo = true
+        
+        // TODO: Use constants
+        let messageViewMarginRight: CGFloat = 80
+        let messageLeftMargin: CGFloat = 48
```

Comment:
Unimportant, but it would be nice if these names had the same format. I'm also wondering if we should be saying `leading` and `trailing` when defining such variables or adding IBOutlets for constraints?

Reply:
Yes I'll remove local constant and use proper global constants in a future PR (https://github.com/vector-im/element-ios/issues/5409)